utils/merge_nanoAOD_files.py

```diff
@@ -0,0 +1,67 @@
+import glob
```

Comment:
Is this based on the LbL script? Do you think we could make it more general and a part of tea?

Reply:
Yeah I guess, we also have merge_histograms in ttalps utils